src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -260,19 +261,23 @@ def main(
     """
     # Temporary warning, to be removed
     if model_args.tie_word_embeddings is True:
-        _LOGGER.warning(
+        logger.warning(
             "The tie_word_embeddings flag is by default set to False. "
             "This guarantees that the one-shot algorithm saves the final "
             "weights without errors. Detected tie_word_embeddings=True. "
             "This may cause issues with the one-shot algorithm on save. "
         )
-    # Setup logging
-    log_level = training_args.get_process_log_level()
-    _LOGGER.setLevel(log_level)
-    datasets.utils.logging.set_verbosity(log_level)
-    transformers.utils.logging.set_verbosity(log_level)
-    transformers.utils.logging.enable_default_handler()
-    transformers.utils.logging.enable_explicit_format()
+
+    # Setup logger
+    # TODO: taking this out for now, these aren't valid calls and I was unsure what
+    # the intended purpose was originally
+    #
+    # log_level = training_args.get_process_log_level()
+    # logger.set_verbosity(log_level)
+    # datasets.utils.logger.set_verbosity(log_level)
+    # transformers.utils.logger.set_verbosity(log_level)
+    # transformers.utils.logger.enable_default_handler()
+    # transformers.utils.logger.enable_explicit_format()
```

Comment:
@markurtz these weren't valid function calls, I couldn't tell what the purpose of this code block was so commenting it out for now, it was breaking a bunch of tests and the logging seems to work fine without it

Reply:
Noted elsewhere, but removing this completely and we'll cover it during the audit of the transformer pipeline pathways